node.go

```diff
@@ -135,53 +53,65 @@ func (n *SphinxNode) PrefixFreeDecode(s []byte) (int, []byte, []byte) {
 	return Failure, nil, nil
 }
 
-// Unwrap unwraps a layer of encryption from a sphinx packet
-// and upon success returns an UnwrappedMessage, otherwise an error.
-func (n *SphinxNode) Unwrap(packet *SphinxPacket) (*UnwrappedMessage, error) {
+// UnwrappedMessage is produced by SphinxNode's Unwrap method
+type UnwrappedMessage struct {
+	ProcessAction int
+	Alpha         []byte
```

Comment:
the sphinx paper and sphinx reference code uses alpha, beta, gamma, delta.
but this could be replaced with:
ephemeral key
routing info
MAC
body